### PR TITLE
hello world

### PR DESCRIPTION
**Pull request by lichab**
hello world feature added

### CODE REVIEW

File: staff/lisandro-bertoli/hello-world/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<meta http-equiv="X-UA-Compatible" content="ie=edge" />
+		<link rel="stylesheet" href="style.css" />
+		<title>Hello World</title>
+	</head>
+	<body>
+		<h1>Hello, World!</h1>
```
Comment (lichab) [inline staff/lisandro-bertoli/hello-world/index.html]: blah blah blah

File: staff/lisandro-bertoli/hello-world/style.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+body {
+	background-color: yellow;
+}
+
+h1 {
+	cursor: pointer;
```
Comment (lichab) [inline staff/lisandro-bertoli/hello-world/style.css]: blah blah blah

File: staff/lisandro-bertoli/hello-world/style.css [hunk 3]
```diff
@@ -0,0 +1,7 @@
+body {
+	background-color: yellow;
```
Comment (alexmj-webdev) [inline staff/lisandro-bertoli/hello-world/style.css]: una obra maestra

File: staff/lisandro-bertoli/hello-world/index.html [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+		<p>Eres el mas Vurro</p>
```
Comment (alexmj-webdev) [inline staff/lisandro-bertoli/hello-world/index.html]: hey, this text should be <p>Burro</p>

**Pull request merged**